tox.ini

```diff
@@ -11,4 +12,4 @@ deps =
     wptserve >= 1.4.0
```

Comment:
manifestparser and moztest are no longer used now given that `test.py` got deleted. So please also kill those dependencies.